pkg/standalone/uninstall.go

```diff
@@ -44,19 +46,33 @@ func removeContainers(uninstallAll bool, dockerNetwork string) []error {
 				containerErrs,
 				fmt.Errorf("could not remove %s container: %s", DaprRedisContainerName, err))
 		}
+		fmt.Println("trying to remove zipkin container: ", DaprZipkinContainerName)
+		_, err = utils.RunCmdAndWait(
+			"docker", "rm",
+			"--force",
```

Comment:
This code seems to be repeated. Could we extract a method and pass in the container name to remove?